workloads/network-perf/csv_gen.py

```diff
@@ -191,6 +191,6 @@ def push_to_gsheet(csv_file_name, google_svc_acc_key, email_id):
 generate_csv(params.yaml_files, f"{params.sheetname}.csv")
 if "EMAIL_ID_FOR_RESULTS_SHEET" in os.environ:
     push_to_gsheet(
-        f"{params.sheetname}.csv", "gsheet_key.json", os.environ["EMAIL_ID_FOR_RESULTS_SHEET"],
+        f"{params.sheetname}.csv", "/root/.secrets/gsheet_key.json", os.environ["EMAIL_ID_FOR_RESULTS_SHEET"],
```

Comment:
Maybe change this to look for secrets under $HOME/.secrets instead of defaulting to root user, also being able to define the path to the secrets using a var/flag should make it more flexible for automation. Thoughts?

Reply:
done